features/command_line_interface/options.feature

```diff
@@ -41,7 +41,9 @@ Feature: Reek can be controlled using command-line options
 
       Configuration:
           -c, --config FILE                Read configuration options from FILE
-              --smell SMELL                Detect smell SMELL (default: all enabled smells)
+              --smell SMELL                Only look for a specific smell.
+                                           Call it like this: reek --smell PrimaDonnaMethod source.rb
+                                           Check out https://github.com/troessner/reek/blob/master/docs/Code-Smells.md for a list of smells
```

Comment:
Why? It is helpful, isn't it? And it's not like the output for our help flag is a wall of text already.
In doubt, I would **always** pick the option that is more helpful for the user.